### PR TITLE
Add support for the monitoring operator

### PR DESCRIPTION
**Pull request by razvan**
Fixes: #26

### CODE REVIEW

File: stackable-scripts/apply-cr.sh [hunk 1]
```diff
@@ -7,9 +7,13 @@ case $CR_NAME in
         kubectl apply -f /stackable-scripts/cr/repository.yaml
         ;;
     zookeeper-cluster)
+      kubectl apply -f /stackable-scripts/cr/zookeeper-cluster.yaml
         # Get the host name of the first node (and remove the "node/" prefix from the kubectl output)
-        HOSTNAME=$(kubectl get nodes -o name | head -1 | awk '{host=substr($0, 6); print host}')
-        cat /stackable-scripts/cr/zookeeper-cluster.yaml | sed "s/{{HOSTNAME}}/$HOSTNAME/g" | kubectl apply -f -
+#        HOSTNAME=$(kubectl get nodes -o name | head -1 | awk '{host=substr($0, 6); print host}')
+#        cat /stackable-scripts/cr/zookeeper-cluster.yaml | sed "s/{{HOSTNAME}}/$HOSTNAME/g" | kubectl apply -f -
```
Comment (maltesander) [inline stackable-scripts/apply-cr.sh:13]: Can you delete that?

File: stackable-scripts/spec/monitoring-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                  type: string
+              type: object
+          required:
+            - spec
+          title: MonitoringCluster
+          type: object
+      served: true
+      storage: true
+      subresources:
+        status: {}
```
Comment (maltesander) [inline stackable-scripts/spec/monitoring-cluster.yaml:211]: Use the CRDs provided in the packages?

File: stackable-scripts/spec/zookeeper-cluster.yaml [hunk 3]
```diff
@@ -68,13 +158,12 @@ spec:
...
+                          - replicas
                           - selector
                         type: object
                       type: object
                   required:
-                    - selectors
+                    - roleGroups
                   type: object
                 version:
                   enum:
```
Comment (maltesander) [inline stackable-scripts/spec/zookeeper-cluster.yaml:169]: Use the CRD provided in the packages?

**Review by maltesander — CHANGES_REQUESTED** [review]
Just small changes.

**Comment by razvan** [comment]
Tested with debian and agent, monitoring-operator, zookeeper-operator, kafka-operator and spark-operator.

centos testing is blocked by https://github.com/stackabletech/monitoring-operator/pull/5

**Review by maltesander — APPROVED** [review]
LGTM

**Pull request merged**